index.html

```diff
@@ -4,15 +4,206 @@
+             </ul>
+        </div>
+        <div class="work-button">
+            <button>See Project</button>
+        </div>
+    </section>
+
+    <section class="work-section">
+        <div class="work-imgs card4">
+            <div class="work-bg"></div>
+        </div>
+        <div class="work-topic">
+            <h2>Tonic</h2>
+        </div>
+        <div class="work-details">
+            <ul>
+                <li>∘ CANOPY</li>
+                <li class="work-2nd">∘ BACK END DEV</li>
+                <li class="work-2nd">∘ 2023</li>
+            </ul>
+        </div>
+        <div class="work-paragraph">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-langs">
+             <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+             </ul>
+        </div>
+        <div class="work-button">
+            <button>See Project</button>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-heading">
+            <h2>About Myself</h2>
+        </div>
+        <div class="about-description">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="about-connect">
+            <h3><a href="">LET’S CONNECT</a></h3>
+        </div>
+        <div class="about-socials">
+            <div class="social-bar">
+                <ul>
+                    <li><a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a></li>
+                    <li><a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedin.png" alt="LinkedIn logo"></a></li>
+                    <li><a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a></li>
+                    <li><a href="#" class="connect"><img src="image-resources/M.png" alt=" logo"></a></li>
+                    <li><a href="#" class="connect"><img src="image-resources/new.png" alt=" logo"></a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="about-button">
+            <button>Get my resume</button>
+        </div>
+        <div class="about-langs">
+            <h2>Languages <span class="arrow-down"><i class="fa-solid fa-caret-down"></i></span></h2>
+            <ul>
+                <li><a href="#"><i class="fa-brands fa-square-js"></i> Javascript</a></li>
+                <li><a href="#"><i class="fa-brands fa-html5"></i> HTML</a></li>
+                <li><a href="#"><i class="fa-brands fa-css3-alt"></i> CSS</a></li>
```

Comment:
- **[OPTIONAL]** As suggested in the project requirement to follow the design system as much as possible I highly recommend using the assets from the Figma design for the skills section.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219605452-9511ab1a-57cd-426d-8619-986e9e2191b1.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219605407-77e3f523-acd3-4be8-afbb-44599d4449a4.png)